ar.com.softwareperonista.Rockarrolla.json

```
@@ -0,0 +1,77 @@
+{
+    "app-id" : "ar.com.softwareperonista.Rockarrolla",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "rockarrolla",
+    "branch" : "stable",
+    "finish-args" : [
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--device=dri"
+    ],
+    "build-options" : {
+        "cflags" : "-O2 -g",
+        "cxxflags" : "-O2 -g",
+        "env" : {}
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "doctrina",
+            "buildsystem" : "meson",
+            "builddir" : true,
+            "config-opts" : [
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://gitlab.com/softwareperonista/doctrina/-/archive/0.1.13.2/doctrina-0.1.13.2.tar.gz",
+                    "sha256" : "d9863cf3a3eddbd5738509b434a949ae2a6e66de9b6687505d62cd767fe27955"
+                }
+            ]
+        },
+        {
+            "name" : "taglib",
+            "buildsystem" : "cmake",
```

```suggestion
            "buildsystem" : "cmake-ninja",
```

unless it doesn't work